src/seals/diagnostics/proc_goal.py

```diff
@@ -0,0 +1,62 @@
+"""Gridworld with large random initial state."""
```

Comment:
```suggestion
"""Large gridworld with random agent and goal position."""
```
I don't really get what a large initial state means -- is it larger than subsequent states? :)